containers.go

```diff
@@ -647,13 +647,25 @@ func (d *Docker) CreateDownloadContainer(job *model.Job, input *model.StepInput,
 	hostConfig := &container.HostConfig{}
 	invID := job.InvocationID
 
-	image = d.cfg.GetString("porklock.image")
-	tag = d.cfg.GetString("porklock.tag")
+	image = d.cfg.GetString("porklock.image")    // Used to pull the correct porklock image.
+	tag = d.cfg.GetString("porklock.tag")        // Used to pull the correct porklock image.
+	vaultaddress := d.cfg.GetString("vault.url") // Used by porklock to pull the iRODS config from Vault.
+	vaulttoken := d.cfg.GetString("vault.token") // Used by porklock to pull the iRODS config from Vault.
 
+	// The porklock image is needed before a container can be created from it.
 	if err = d.PorkPull(); err != nil {
 		return "", err
 	}
 
+	// It makes little sense to have only one of these set
```

Comment:
I've thought about that, too. Right now porklock will fail outright if one or the other isn't set, so the job will still fail. I'm undecided about whether or not that's better, but it's something we can change pretty easily if it becomes an issue in the future.